packages/processor/src/codegen/Parser.ts

```diff
@@ -74,7 +74,7 @@ export default class Parser extends BaseParser {
 
         const result: EnumType = {
           type: 'enum',
-          name: uncapitalize(type.lookupName),
+          name: uncapitalize(type.lookupName).replace('IngressEgress', this.getPalletName()),
```

Comment:
the fix

Reply:
```suggestion
          name: uncapitalize(type.lookupName).replace(/^IngressEgress/, this.getPalletName()),
```
is it necessary to make it a bit more strict maybe?

Reply:
no this should be ok, the name looks like `PalletCfIngressEgressDepositFailedDetails` and we just need to insert the chain in there somewhere to make it distinct

Reply:
the pallet name is like `SolanaIngressEgress`, so it just adds the chain name